docs/application/dotnet/guides/security/requesting-permissions.md

```diff
@@ -76,14 +76,14 @@ To check whether an application has permission to use a privilege, and to reques
 
         The dialog box asking for user permission is shown only if the `RequestPermission()` method does not throw an exception.
 
-        ```
+        ```csharp
                     case CheckResult.Ask:
                         PrivacyPrivilegeManager.RequestPermission(cameraPrivilege);
                         break;
                 }
         ```
 
-    ```
+    ```csharp
```

Comment:
It was not aligned because  comment in lines 73-77 was only about one switch case (lines 80-83) and not about the rest lines (87+).


`catch` statement is a part of `try & catch` block. 

```csharp
try {
// do something that may result in exception
}
catch (Exception ex) { 
// handle exceptions that occurred in try block
}
```

Reply:
Thank you very much. 
`catch` statement is a part of `try & catch` block. I completely agree with your explanation.
 I was suggesting that if the alignment is not proper it can confuse the end user.

Reply:
knoxid: e.borowski